api/provider/aws/deploy_delete.go

```diff
@@ -37,24 +38,14 @@ func (d *DeployProvider) deleteDeploy(taskARN string) error {
 	}
 
 	if _, err := d.AWS.ECS.DeregisterTaskDefinition(input); err != nil {
+		log.Printf("[WARN] Deploy not found\n")
```

Comment:
this should be printed after we check the error below. Also, only use `log` to print to stdout (maybe this is leftover debug code?)

Reply:
This was actually a debug leftover, so will remove --- is it necessary to print this to logs given the error will already be present?